pages/components/attention.html

```diff
@@ -79,63 +79,135 @@ <h2 class="mt-24 mb-16">Props</h2>
               </td>
               <td>false</td>
             </tr>
+            <tr>
+              <td>can-close</td>
+              <td>
+                <div>boolean</div>
+                <div class="annotation">Render Attention element with a close button.</div>
+              </td>
+              <td>false</td>
+            </tr>
+            <tr>
+              <td>distance</td>
+              <td>
+                <div>number</div>
+                <div class="annotation">The distance from which to offset the attention element from the target element vertically.</div>
+              </td>
+              <td>8</td>
+            </tr>
+            <tr>
+              <td>skidding</td>
+              <td>
+                <div>number</div>
+                <div class="annotation">The distance from which to offset the attention element along its target element horizontally.</div>
+              </td>
+              <td>0</td>
+            </tr>
+            <tr>
+              <td>flip</td>
+              <td>
+                <div>boolean</div>
+                <div class="annotation">Whether Attention element should flip its placement in order to keep it in view.</div>
+              </td>
+              <td>false</td>
+            </tr>
+            <tr>
+              <td>fallbackPlacements</td>
```

Comment:
```suggestion
              <td>fallback-placements</td>
```

Reply:
Updated in this commit:[150b818]( https://github.com/warp-ds/elements/pull/135/commits/150b8180215bfa0fac4cf9f3983db2c23d8650a7)